src/pages/UsersPage/UsersPageContainer.tsx

```diff
@@ -1,11 +1,11 @@
 import axios from 'axios';
 import React, { useState, useEffect } from 'react';
-import { IUser, IUserWithCheck } from './interfaces';
+import { IUser } from './interfaces';
 import { UsersPage } from './UsersPage';
 
 export const UsersPageContainer: React.FC = () => {
   const [users, setUsers] = useState([] as IUser[]);
-  const [usersWithCheck, setUsersWithCheck] = useState([] as IUserWithCheck[]);
+  const [usersSet, setUsersSet] = useState(new Set());
```

Comment:
Don't use data type in the name. Rename with checkedUsersIDs, checkedIDs, checkedUsers or somethig else

Reply:
noted, done